src/ToDoList.jsx

```diff
@@ -0,0 +1,23 @@
+export default function ToDoList({ todos, onClick }) {
+  if (todos) {
+    return (
+      todos.map((todo, id) => (
+        <li key={todo.id}>
+          <p>
+            {id}
+            {todos}
+          </p>
+          <button
+            type="button"
+            onClick={onClick(id)}
+          >
+            완료
+          </button>
+        </li>
+      ))
+    );
+  }
+  if (!todos) {
+    return <p> 할일이 없어요!</p>;
+  }
```

Comment:
```suggestion
  if (!todos) {
    return <p> 할일이 없어요!</p>;
  }
  
  return (
    todos.map((todo, id) => (
      <li key={todo.id}>
        <p>
          {id}
          {todos}
        </p>
        <button
          type="button"
          onClick={onClick(id)}
        >
          완료
        </button>
      </li>
    ))
  );
}
```

[guard clauses](https://www.refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html) 형태로 표현해주면 이런 식이 될거에요.

예외적인 상황을 먼저 if 문으로 처리해주고 일반적인 상황을 return 해주는 방식이에요 ;)

Reply:
네 알겠습니다! 감사합니다